timed/employment/views.py

```diff
@@ -345,6 +345,18 @@ class LocationViewSet(ReadOnlyModelViewSet):
     serializer_class = serializers.LocationSerializer
     ordering = ("name",)
 
+    def get_queryset(self):
+        """Don't show locations to customers."""
+        user = self.request.user
+
+        queryset = models.Location.objects.all()
+
+        try:
+            if models.Employment.objects.get_at(user=user, date=datetime.date.today()):
+                return queryset
+        except models.Employment.DoesNotExist:
+            return queryset.none()
+
```

Comment:
This pattern is used all over the place.

This try/if/except construct also is rather verbose. I suggest adding a method `has_active_employment()` to the user model (or a `get_active_employment()` and make it return `None` instead of raising an exception).
This would simplify a ton of call sites